modules/base/markdown.go

```diff
@@ -250,6 +250,7 @@ func PostProcessMarkdown(rawHtml []byte, urlPrefix string) []byte {
 func RenderMarkdown(rawBytes []byte, urlPrefix string) []byte {
 	result := RenderRawMarkdown(rawBytes, urlPrefix)
 	result = PostProcessMarkdown(result, urlPrefix)
+	Sanitizer.AllowAttrs("class").Matching(regexp.MustCompile(`[\p{L}\p{N}\s\-_',:\[\]!\./\\\(\)&]*`)).OnElements("code")
```

Comment:
Yeah, `Sanitizer` is a global variable, but since `OnElements` has return value, I suppose it has to be re-assign back to `Sanitizer`?

Also, shouldn't only do this once instead of repeating whenever renders a markdown?

Not sure about how `bluemonkey` works, I can be wrong.